packages/devextreme/js/__internal/scheduler/m_compact_appointments_helper.ts

```diff
@@ -181,4 +184,29 @@ export class CompactAppointmentsHelper {
       .append($('<span>').text(text))
       .addClass(APPOINTMENT_COLLECTOR_CONTENT_CLASS);
   }
+
+  _localizeDate(date) {
+    return `${dateLocalization.format(date, 'monthAndDay')}, ${dateLocalization.format(date, 'year')}`;
+  }
+
+  _getStartDate(appointment) {
```

Comment:
@tongsonbarbs what do you think if you merge `_getStartDate` and `_getEndDate` into one function which receives `appointment.date`, looks like both of these functions are doing the same job.

Reply:
I wanted to set a practice where we follow Single Responsibility Pattern.. These both functions may look similar but they return different and specific data.